support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -254,6 +255,20 @@ function preventDefaultValidityMessage(
 	}
 }
 
+type ChangeButtonProps = {
+	geoId: GeoId;
+};
+
+function ChangeButton({ geoId }: ChangeButtonProps) {
+	return (
+		<a href={`/${geoId}/contribute`}>
```

Comment:
Looks good, tested this and appeared to raise a `navigateWithPageView` Ophan event.

Reply:
I think it would just raise a `pageview` event as this uses standard browser navigation.